### PR TITLE
Fix gallery view compilation error for SlackClone

### PR DESCRIPTION
**Pull request by SagarSDagdu**
### What does this PR do

This PR fixes the SlackClone app's errors, along with the snapshot tests that were failing due to the removal of create channel button from the UI

### CODE REVIEW

**Review by nuno-vieira — APPROVED** [review]
LGTM!

**Review by skorepak — APPROVED** [review]

File: Sources/StreamChatUI/Utils/ComponentsProvider_Tests.swift [hunk 1]
```diff
@@ -101,7 +101,7 @@ private class TestViewWithExtraData<ExtraData: ExtraDataTypes>: UIView, Componen
 
 private class TestViewWithExtraDataController<ExtraData: ExtraDataTypes>: UIViewController, ComponentsProvider {
     let subView = TestViewWithExtraData<ExtraData>()
-    lazy var TestCreateChannelButton = components.createChannelButton.init()
+    lazy var TestCreateChannelButton = components.galleryView.init()
```
Comment (evsaev) [inline Sources/StreamChatUI/Utils/ComponentsProvider_Tests.swift:104]: let's make it `TestGalleryView` so the naming reflect the component type

**Review by evsaev — APPROVED** [review]
Great job! Thanks for fixing it 👍

File: Sources/StreamChatUI/Utils/ComponentsProvider_Tests.swift (hunk 1 above)
Comment (SagarSDagdu) [reply Sources/StreamChatUI/Utils/ComponentsProvider_Tests.swift:104]: Missed this one, I'll fix this in the next one. Thanks!

**Pull request merged**